lightning/src/chain/chainmonitor.rs

```diff
@@ -655,7 +653,9 @@ where C::Target: chain::Filter,
 					} else {
 						monitor_state.channel_perm_failed.store(true, Ordering::Release);
 					}
-					log_error!(self.logger, "Failed to persist channel monitor update: {:?}", e);
+					log_error!(self.logger, "Failed to persist ChannelMonitor update for channel {}: {:?}", log_funding_info!(monitor), e);
```

Comment:
Looks like the log assertion in `test_monitor_and_persister_update_fail` needs to be updated for this change.

Reply:
Oops.